README.md

```diff
@@ -178,44 +178,44 @@ try {
 // ConditionMatchFailure{condition=(a and b), cause=java.util.concurrent.CompletionException: java.lang.IllegalStateException, async=false, thread=Test worker, delay=0ms, timeout=INF, startTime=1672051528774ms, endTime=1672051528776ms, duration=2ms}
 ```
 
-## Easy to integrate with Spring Framework
+## Kotlin DSL support
+If you are using the [Kotlin programming language](https://kotlinlang.org), try to use _Conditional-Kotlin_. It makes it easier for you to make conditional expressions using the Kotlin DSL.
 
-If you are using [Spring Framework](https://spring.io/projects/spring-framework), you can also make conditional expressions like this:
```

Comment:
Is there a reason for removing usage-related content from Spring Framework?

Reply:
As you know, Conditional can be used in various JVM-based environments.
But if the README mentions Spring Framework, it might seem that Conditional is good to use only in Spring Framework. So, it is better not to specify the environment in which Conditional can be used.

As you said, it would be nice to mention in the README about the examples module and it would be nice to think about where and how to add this. Thank you for your review. 👍